q.py

```diff
@@ -185,11 +185,17 @@ def add(self, items, sep='', wrap=True):
             self.column += size
 
     def __init__(self):
-        self.writer = self.Writer(self.FileWriter(self.OUTPUT_PATH), self.time)
+        time = self.time
+        self.writer = self.Writer(self.FileWriter(self.OUTPUT_PATH), time)
```

Comment:
Hmm, what is this change for?

Reply:
It just caches self.time in time. time is used several times in this method.

Reply:
Oh, just for ease of typing, you mean? Makes sense.